model/dao/TaskDAO/PostgreSQLTaskDAO.php

```diff
@@ -739,14 +739,6 @@ public function partialUpdate(DirtyTaskVO $taskVO) {
      * @return array OperationResult the array of {@link OperationResult} with information about operation status
      */
     public function batchPartialUpdate($tasks) {
-        if (!$this->checkOverlappingWithDBTasks($tasks)) {
-            $result = new OperationResult(false);
-            $result->setErrorNumber(10);
-            $result->setMessage("Task update failed:\nDetected overlapping times.");
-            $result->setResponseCode(400);
-            return array($result);
-        }
```

Comment:
I would keep the validation in the back end too just to make sure this case is never stored in the database. Does the user get duplicated error messages if we keep it here too?

Reply:
Yeah, this could work. The frontend will validate first and if not valid will not hit the backend. So, if for some reason the frontend validation is borked, then the backend would send the same message it always had. We'd want to test and see what it actually does. I will work on that.

Reply:
I tested quite a bit and the front is catching everything I throw at it so far, so I haven't seen what it would look like if a task got through. But it's still a good idea to restore the backend check. I've done so and pushed.